src/clients/MultiCallerClient.ts

```diff
@@ -342,6 +342,11 @@ export class MultiCallerClient {
       // simulated. In this case, drop the aggregation and revert to undefined to force estimation on submission.
       gasLimit = isDefined(gasLimit) && isDefined(txn.gasLimit) ? gasLimit.add(txn.gasLimit) : undefined;
     });
+    const multicallMethod = !simulate
+      ? transactions.every((txn) => ["fillV3Relay", "requestV3SlowFill"].includes(txn.method))
+        ? "tryMulticall"
+        : "multicall"
+      : "multicall";
```

Comment:
Thoughts on consolidating the logic a bit? I think this should be equivalent.

```suggestion
    const method = !simulate && transactions.every(({ method }) => ["fillV3Relay", "requestV3SlowFill"].includes(method))
        ? "tryMulticall"
        : "multicall";
```